packages/frontend/src/components/SchedulerModals/SchedulerModalBase/SchedulerForm.tsx

```diff
@@ -246,4 +366,10 @@ const SchedulerForm: FC<
     );
 };
 
+const SchedulerForm: FC<
+    { disabled: boolean } & React.ComponentProps<typeof Form>
+> = ({ disabled, methods, ...rest }) => {
+    return <SchedulerSettings disabled={disabled} methods={methods} />;
+};
+
```

Comment:
Why do we need this extra component?

Reply:
Because I initially implemented tabs, so I had to split them up, then I kept the component, but I should probably revert back...